### PR TITLE
Adding qPCR values

### PR DESCRIPTION
**Pull request by josenavas**
Forgot to add a couple of columns

### CODE REVIEW

File: amgut/db/patches/0041.sql [hunk 1]
```diff
@@ -479,6 +479,8 @@ CREATE TABLE pm.shotgun_library_prep_kit (
      water_volume_nl              real  NOT NULL,
      shotgun_i5_index_id          bigint  ,
      shotgun_i7_index_id          bigint  ,
+     qpcr_concentration           real ,
```
Comment (antgonza) [inline amgut/db/patches/0041.sql]: just want to confirm that real is fine and not double?

Comment (josenavas) [inline amgut/db/patches/0041.sql]: I would say so - we've been using real for everything. Any concern?

Comment (antgonza) [inline amgut/db/patches/0041.sql]: just making sure ...

Comment (wasade) [inline amgut/db/patches/0041.sql]: real looks like it's comparable to single precision float but I didn't check if its actually ieee 754 compliant or whatever that spec is

**Pull request merged**